bridges/scalajs-1/src/main/scala/bloop/scalajs/JsBridge.scala

```diff
@@ -51,6 +53,41 @@ object JsBridge {
       }
     override def trace(t: => Throwable): Unit = logger.trace(t)
   }
+  private object ScalaJSLinker {
+    private val cache = TrieMap.empty[Path, WeakReference[(JsConfig, Linker)]]
+    def reuseOrCreate(config: JsConfig, target: Path): Linker =
+      if (config.mode == LinkerMode.Release) createLinker(config)
+      else
+        cache.get(target) match {
+          case Some(WeakReference((`config`, linker))) => linker
+          case _ =>
+            val newLinker = createLinker(config)
+            cache.update(target, WeakReference((config, newLinker)))
+            newLinker
+        }
+    private def createLinker(config: JsConfig): Linker = {
+      val isFullLinkJS = config.mode == LinkerMode.Release
+      val semantics =
+        if (isFullLinkJS) Semantics.Defaults.optimized
+        else Semantics.Defaults
+      val scalaJSModuleKind = config.kind match {
+        case ModuleKindJS.NoModule => ScalaJSModuleKind.NoModule
+        case ModuleKindJS.CommonJSModule => ScalaJSModuleKind.CommonJSModule
+        case ModuleKindJS.ESModule => ScalaJSModuleKind.ESModule
+      }
+
+      val useClosure = isFullLinkJS && config.kind != ModuleKindJS.ESModule
+
+      val linkerConfig = StandardConfig()
+        .withOptimizer(true)
+        .withClosureCompilerIfAvailable(useClosure)
```

Comment:
```suggestion
        .withClosureCompiler(useClosure)
```
This is JVM-only code, so this stronger method exists.